first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,630 @@
+    "\n",
+    "convert_to_dc_widget = ConvertToDataCollectionWidget()\n",
+    "display(convert_to_dc_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [],
+    "toc-hr-collapsed": true
+   },
+   "source": [
+    "### Publish an initial version of your data collection\n",
+    "<a id=\"publish-version\"></a>\n",
+    "\n",
+    "Now you'll publish a version of your newly created data collection which can be shared with others and added to other workspaces. \n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to create a version and publish it.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "- `workspace_id`: Must match the workspace ID of the workspace created above.\n",
+    "- `version`: Must be a string. This field is useful in particular if future releases of the data in this data collection are planned. It's suggested that the same versioning strategy be used for all releases of a data collection to make it transparent for users (e.g., if the first release is version `1.0`, the next release should be `2.0`, not `Version 2`).\n",
+    "- `description`: Must be a string. This field should describe this particular version of your data collection and is displayed in the data catalog.\n",
+    "\n",
+    "The output should resemble:\n",
+    "```\n",
+    "Workspace properties successfully updated.\n",
+    "ID:                <STUDY_NAME>_<YYMMDD>_dc_ws\n",
+    "Name:               <STUDY_NAME>-Data-<YYMMDD>\n",
+    "Description:       <DESCRIPTION>\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=<GOOGLE_PROJECT_ID>\n",
+    "Properties:\n",
+    "  terra-workspace-short-description: descriptive content\n",
+    "  terra-workspace-version: 1.0\n",
+    "  terra-type: data-collection\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       Workspace properties successfully updated.\n",
+    "ID:                <STUDY_NAME>_<YYMMDD>_dc_ws\n",
+    "Name:              <STUDY_NAME> Data <YYMMDD>\n",
+    "Description:       <DESCRIPTION>\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=<GOOGLE_PROJECT_ID>\n",
+    "Properties:\n",
+    "  terra-workspace-short-description: <SHORT_DESCRIPTION>\n",
+    "  terra-workspace-version: <VERSION>\n",
+    "  terra-type: data-collection\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       0\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "class PublishVersionWidget(object):\n",
```

Comment:
I think this widget (**and maybe others**) need to quote the description string, I'm getting this error:
 
<img width="760" alt="CleanShot 2023-09-22 at 08 52 26@2x" src="https://github.com/DataBiosphere/terra-axon-examples/assets/115093/1f885781-bd08-481c-b647-f40ef83b8825">

Reply:
Fixed!